cmd/flags_misc.go

```diff
@@ -87,5 +134,72 @@ func ParseMiscFlags(cmd *cobra.Command) error {
 			log.Println(http.ListenAndServe("0.0.0.0:6000", mux))
 		}()
 	}
+
+	ok, err = cmd.Flags().GetBool(tracingFlag)
+	if err != nil {
+		return err
+	}
+
+	if ok {
+		exp, err := otlptracehttp.New(cmd.Context(),
+			otlptracehttp.WithEndpoint(cmd.Flag(tracingEndpointFlag).Value.String()),
+			otlptracehttp.WithCompression(otlptracehttp.GzipCompression),
+			otlptracehttp.WithInsecure(),
```

Comment:
Is this required?

Reply:
Insecure or all options you point to?

Reply:
Do you ask about the Insecure option or all the pointed options?

Reply:
Insecure option

Reply:
Linking https://github.com/celestiaorg/celestia-node/issues/934

Reply:
Yes, it is necessary for a simple setup like we have.

Reply:
If we don't provide that, nothing works until we don't provide custom TLSConfig

Reply:
Given it is possible to configure celestia-node to send metrics to an OTEL Collector over HTTPS and that this will be the convention during the incentivized testnet, I think we should remove this line

```suggestion

```

In a subsequent issue we can add a CLI flag for `--insecure-metrics` and `--insecure-tracing` that enables the `WithInsecure()` option for node operators who want to host their own OTEL Collector and don't want to configure TLS certificates on their OTEL Collector. What do you think?

Reply:
I've implemented tls option for both metrics and tracing and removed Insecure option being the default, so resolving